src/bin/gpfdist/regress/input/exttab1.source

```diff
@@ -456,6 +456,52 @@ format 'csv';
 select count(*) from bad_location2;
 drop external table bad_location2;
 
+--
+-- Test DEFAULT and NOT NULL
+--
+
+-- Test NOT NULL for readable (constraint should be ignored)
+CREATE EXTERNAL TABLE readable_not_null (a int, b int NOT NULL, c int NULL)
+LOCATION ('gpfdist://@hostname@:7070/exttab1/data_with_null.data')
```

Comment:
Is gpfdist required here? Will file:// work? Won't test be simpler and faster if file:// is used?

Reply:
*Here* it isn't required because the table is readable, but for writable tables it is. I decided to not split the test suite and put it all in gpfdist, since the tests are related.